.github/workflows/build_and_test.yml

```diff
@@ -623,7 +628,7 @@ jobs:
       if: fromJSON(inputs.envs).PYSPARK_CODECOV == 'true'
       uses: codecov/codecov-action@v5
       env:
-        CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
+        CODECOV_TOKEN: ${{ inputs.codecov_token }}
```

Comment:
After this change, will `inputs.codecov_token` be in plaintext?

Reply:
Using the current approach, token is no longer plaintext.